client/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.57",
```

Comment:
ich glaube alpha hat damit zu tun https://material-ui.com/components/about-the-lab/